src/weatherDataSource.js

```diff
@@ -0,0 +1,42 @@
+export default class WeatherDataSource {
+    constructor() {
+        this.apiKey = '84af29f03ef35bb4dd7f1e9b0ad575e2';
+    }  
+    getweather(city) { 
```

Comment:
Camelcase naming convention would want this function to be named getWeather instead

Reply:
I changed to getWeather